sos/report/plugins/containers_common.py

```diff
@@ -17,11 +18,33 @@ class ContainersCommon(Plugin, RedHatPlugin, UbuntuPlugin):
     plugin_name = 'containers_common'
     profiles = ('container', )
     packages = ('containers-common', )
+    option_list = [
+        ('rootlessusers', 'colon-separated list of users\' containers info',
+         '', ''),
+    ]
 
     def setup(self):
         self.add_copy_spec([
             '/etc/containers/*',
             '/usr/share/containers/*',
+            '/etc/subuid',
+            '/etc/subgid',
         ])
+        self.add_cmd_output(['loginctl user-status'])
+
+        users_opt = self.get_option('rootlessusers')
+        users_list = []
+        if users_opt and users_opt != '':
```

Comment:
Isn't this a redundant conditional? `''` should return `False` anyways.

Reply:
Ah right, too blind copy&paste&modify code :)